editors/linux-wps-office/files/patch-opt_kingsoft_wps-office_desktops_wps-office-et.desktop

```diff
@@ -0,0 +1,11 @@
+--- opt/kingsoft/wps-office/desktops/wps-office-et.desktop.orig	2021-06-05 11:29:11 UTC
++++ opt/kingsoft/wps-office/desktops/wps-office-et.desktop
+@@ -1,7 +1,7 @@
+ [Desktop Entry]
+ Comment=Use WPS Spreadsheets to analyze manage data.
+ Comment[zh_CN]=使用WPS表格分析、管理数据
+-Exec=/usr/bin/et %F
++Exec=et %F
```

Comment:
Maybe `%%PREFX%%/bin/et` and replace %%PREFIX%% with ${PREFIX} in post-patch: ?

I don't think it's very necessary but I personally would prefer this.

Reply:
I actually think that replacing `/usr/bin` with `${PREFIX}` should be enough.